ui/app/components/client-subnav.js

```diff
@@ -1,5 +1,8 @@
 import Component from '@ember/component';
 import { tagName } from '@ember-decorators/component';
+import { inject as service } from '@ember/service';
 
 @tagName('')
-export default class ClientSubnav extends Component {}
+export default class ClientSubnav extends Component {
+  @service keyboard;
```

Comment:
`suggestion` `non-blocking`: I think Ember supports [injecting services directly into templates](https://github.com/emberjs/rfcs/issues/543) so we can remove backing Component classes when they're not needed like here.